tools/st2-setup-samples.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
```

Comment:
Suggestion: only install tests pack, and leave installing examples up to the tests, as needed.